components/window/actions.md

```diff
@@ -75,9 +76,12 @@ You can create a custom action icon and you must provide its `OnClick` handler.
 ````CSHTML
 Custom actions can call C# directly
 
-<TelerikWindow Visible="true">
+<link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.1/css/all.min.css" rel="stylesheet" />
+
+<TelerikWindow Visible="@WindowVisible">
 	<WindowActions>
-		<WindowAction Name="MyAction" Icon="@SvgIcon.Gear" OnClick="@MyCustomActionHandler" />
+		<WindowAction Name="MyAction" Title="MyAction" Icon="@SvgIcon.Gear" OnClick="@MyCustomActionHandler" />
+		<WindowAction Name="CustomClose" Title="CustomClose" Icon="@("fas fa-x")" OnClick="@(()=> WindowVisible = false)" />
```

Comment:
With the current implementation, the `Name` parameter of the `Window` is practically redundant when it comes to custom actions. In other words, it is currently used as a weakly typed enumeration, as the available values are `Close`, `Maximize`, and `Minimize`. 

Do you see value in mentioning this, and perhaps even omitting the `Name` declaration from custom actions?

Reply:
Agreed, I'll omit the `Name` declaration.